example.env

```diff
@@ -1,11 +1,10 @@
-INSTANCE=Development
-PROJECT_TITLE='First Responder COVID-19 Testing - Development'
-PID=8218
-SMTP_SERVER=smtp.ufl.edu
-TIME_ZONE='America/New_York'
-EMAIL_CC=cpb@ufl.edu lawjames1@ufl.edu 
+INSTANCE=Production
+PROJECT=First Responder COVID-19 Testing - Production
+EMAIL_TO=you@example.org
+SET_SEND_SURVEY_EMAIL_TO=you@example.org
```

Comment:
```suggestion
```

Please don't make new variables like this or custom code to consume them. The better solution is to use two configuration files, named for the scripts that consume them if need be. I'll note the corresponding change in the code.